.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature Request 💡
```

Comment:
Discussion: Is it worth splitting out generic feature requests vs module requests?

Reply:
Or maybe we can just modify this to be:

```suggestion
name: Feature / Module Request 💡
about: Suggest a new idea of module for the project.
```

Reply:
On reflection, I think I like the idea of a separate module request section more than combining this into one

Reply:
I've discovered a super major blocker with having two separate templates...what emoji do we use for the module request one?

Reply:
📦

Reply:
I like it